e2e/testsuite/testsuite.go

```diff
@@ -518,8 +518,8 @@ func (s *E2ETestSuite) InitiateChannelUpgrade(ctx context.Context, chain ibc.Cha
 }
 
 // GetIBCToken returns the denomination of the full token denom sent to the receiving channel
-func GetIBCToken(fullTokenDenom string, portID, channelID string) transfertypes.DenomTrace {
-	return transfertypes.ParseDenomTrace(fmt.Sprintf("%s/%s/%s", portID, channelID, fullTokenDenom))
+func GetIBCToken(fullTokenDenom string, portID, channelID string) transfertypes.Denom {
```

Comment:
Just a thought, will this change impact compatability tests based on ibc-go version? 

I'm assuming it should be fine as its just used for creating the `ibc/{hash}` denom.

Reply:
No functional change